### PR TITLE
Update js packages and Correct consts use

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is regularly scheduled js package updates.

This also switches the js const `TokenPerCapacity` to a function and uses the const from the chain instead.

### CODE REVIEW

File: e2e/messages/addIPFSMessage.test.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ describe('Add Offchain Message', function () {
     const f = ExtrinsicHelper.addIPFSMessage(keys, schemaId, ipfs_cid_64, ipfs_payload_len);
     const { target: event } = await f.fundAndSend(fundingSource);
 
-    assert.notEqual(event, undefined, 'should have returned a MessagesInBlock event');
+    assert.notEqual(event, undefined, `should have returned a MessagesInBlock event, got: ${event?.toHuman()}`);
```
Comment (wilwade) [inline e2e/messages/addIPFSMessage.test.ts:108]: Small improvement in the error as I had this fail randomly on me once.

File: e2e/scaffolding/helpers.ts [hunk 2]
```diff
@@ -52,6 +52,11 @@ export const CENTS = 1000000n;
 export const DOLLARS = 100n * CENTS;
 export const STARTING_BALANCE = 6n * CENTS + DOLLARS;
 
+export function getTokenPerCapacity(): bigint {
+  // Perbil
+  return 1_000_000_000n / ExtrinsicHelper.api.consts.capacity.capacityPerToken.toBigInt();
```
Comment (aramikm) [inline e2e/scaffolding/helpers.ts:57]: My guess is that **capacityPerToken** should be `20_000_000`  to make it equal to the old value. Is that correct?

**Review by aramikm — APPROVED** [review]

File: e2e/scaffolding/helpers.ts (hunk 2 above)
Comment (wilwade) [reply e2e/scaffolding/helpers.ts:57]: Correct!

**Review by shannonwells — APPROVED** [review]
ship it

File: e2e/scaffolding/helpers.ts (hunk 2 above)
Comment (wilwade) [reply e2e/scaffolding/helpers.ts:57]: 2% Per billion = `20_000_000`
image

Comment (JoeCap08055) [inline e2e/scaffolding/helpers.ts:57]: Is this token per capacity, or Planck per capacity? Or am I reading this wrong?

Comment (saraswatpuneet) [reply e2e/scaffolding/helpers.ts:57]: You have a billion tokens

**Review by saraswatpuneet — APPROVED** [review]
Lgtm

File: e2e/scaffolding/helpers.ts (hunk 2 above)
Comment (wilwade) [reply e2e/scaffolding/helpers.ts:57]: It is capacity per token (from the chain) in parts per billion (thus the 1 billion) and it is in planks.

**Review by enddynayn — APPROVED** [review]
👍 nice!

**Pull request merged**